### PR TITLE
Add backtrace printing for uncaught exceptions

### PR DESCRIPTION
**Pull request by LaszloLango**
Print backtrace of an uncaught exception by default in debug build.
This feature is not enabled by default in release builds.

IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: tools/build.py [hunk 1]
```diff
@@ -343,7 +347,16 @@ def build_iotjs(options):
 
     # --jerry-debugger
     if options.jerry_debugger:
-        cmake_opt.append('-DFEATURE_DEBUGGER=ON')
+        cmake_opt.append("-DFEATURE_DEBUGGER=ON")
```
Comment (zherczeg) [inline tools/build.py]: What is the change in this line?

Comment (LaszloLango) [reply tools/build.py]: Single quote to double quote to be more consistent. It is unrelated, so I can revert is you want.

Comment (zherczeg) [reply tools/build.py]: No need. I haven't noticed it.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**